### DISCUSSION TITLE
Gradle legacy plugin application instead of plugins DSL

### DISCUSSION THREAD

**Discussion by fondesa** [androiddevelopersitalia/aditogether#13] [Ideas]
Since we just started, we are applying plugin using the new plugins DSL.
I recommend switching to the legacy plugin application because it's more flexible and doesn't require any workaround to solve issues like this one: https://github.com/gradle/gradle/issues/20084

For example, without the workarounds suggested in the issue linked above, we can't depend on the kotlin plugin in `build-tools` having the kotlin plugin applied with new convention also in other projects.

**Comment by fondesa** [comment]
@4face-studi0 @mcatta

**Reply by fardavide** [reply]
All good for me.
I'd suggest having a minimal PR to make the changes clear to everyone

**Reply by fondesa** [reply]
PR with the proposal: https://github.com/androiddevelopersitalia/aditogether/pull/16
